runtime/engine/src/test/java/io/aklivity/zilla/runtime/engine/test/internal/schema/TestSchemaHandler.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc.
+ *
+ * Aklivity licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.aklivity.zilla.runtime.engine.test.internal.schema;
+
+import io.aklivity.zilla.runtime.engine.config.SchemaConfig;
+import io.aklivity.zilla.runtime.engine.schema.SchemaHandler;
+
+public class TestSchemaHandler implements SchemaHandler
+{
+    public TestSchemaHandler(
+        SchemaConfig schema)
+    {
+    }
+
+    @Override
+    public void configure()
+    {
+    }
+
+    @Override
+    public void schemaRegistryClient()
+    {
+    }
+
+    @Override
+    public int registerSchema(
+        String schemaString,
+        String subject,
+        int version,
+        int id)
+    {
+        return 1;
+    }
+
+    @Override
+    public <T> T getSchemasBySubjectAndId(
+        String subject,
+        int id)
+    {
+        return null;
+    }
+
+    @Override
+    public <T> T getSchemasById(
+        int id)
+    {
+        return null;
+    }
+}
```

Comment:
This needs some test behavior for us to try out test validators / encoders / decoders, say in the `kafka` binding for example.